reconcile/openshift_base.py

```diff
@@ -69,7 +71,8 @@ def init_specs_to_fetch(ri: ResourceInventory, oc_map: OC_Map,
                 continue
 
             cluster = namespace_info['cluster']['name']
-            oc = oc_map.get(cluster)
+            privileged = namespace_info.get("clusterAdmin", False)
```

Comment:
since this is a graphql response, the default value will not be returned even if `clusterAdmin` is not defined. the returned value will be None.
you'd probably want `namespace_info.get("clusterAdmin") or False`

Reply:
yes i fell for this every time :) but it is already fixed in 3bb6e27aab73df04125eb94ea7dd3668cd44b16a